bug: codex login status가 stderr로 출력되어 active-codex auth 가드가 항상 차단

## 증상

`make 시작` 실행 시 `agent-loop-active-codex-runner` 서브 스텝이 항상 `Error 1`로 차단되어 active-loop 진입 불가.

\`\`\`
[active-codex-runner] auth check returned blocker even with rc=0
make[1]: *** [agent-loop-active-codex-runner] Error 1
make: *** [agent-loop-active-start] Error 2
\`\`\`

## 원인

[`scripts/agent_loop.py:11954`](scripts/agent_loop.py#L11954) `_active_codex_auth_check`가 `codex login status`의 **stdout만** 검사:

\`\`\`python
if "Logged in using ChatGPT" not in stdout:
    return summary, [f"Codex auth mode requires ChatGPT login; got: {summary}"], []
\`\`\`

실측한 `codex-cli 0.132.0`은 "Logged in using ChatGPT"를 **stderr**로 출력:

\`\`\`bash
$ codex login status 1>/dev/null 2>&1; echo "rc=$?"
rc=0
$ codex login status 1>/tmp/o 2>/tmp/e; cat /tmp/o; echo "---"; cat /tmp/e
---
Logged in using ChatGPT
\`\`\`

결과: rc=0이고 메시지도 정상이지만 stdout이 비어 있어 항상 blocker가 추가됨. state JSON에 `auth_status: "Logged in using ChatGPT"`와 `blockers: ["Codex auth mode requires ChatGPT login; got: Logged in using ChatGPT"]`가 공존하는 모순으로 확인.

## 회귀 소스

오늘(2026-05-28) 머지된 PR #1646 (커밋 1b370e2 "chore: guard active Codex auth")이 신규 추가한 가드 함수. 단위 테스트는 `stdout="Logged in using ChatGPT\n"`로 모킹되어 통과했으나 실제 CLI 동작과 불일치.

## 즉시 우회

\`\`\`bash
make 시작 ACTIVE_CODEX_AUTH_MODE=any
\`\`\`

가드 자체를 skip — ChatGPT/API key 구분 못 함.

## 수정안

`scripts/agent_loop.py:11954`의 stdout 검사를 `combined`(이미 동일 함수 11950 라인에서 stdout+stderr로 계산되어 summary 추출에 사용 중)로 변경. stderr 시나리오 회귀 테스트 추가.

--- a/scripts/agent_loop.py
+++ b/scripts/agent_loop.py
@@ -11951,7 +11951,7 @@ def _active_codex_auth_check(
     summary = _sanitize_inline_text(combined.splitlines()[0] if combined.splitlines() else f"rc={rc}")
     if rc != 0:
         return summary, [f"codex login status failed for ChatGPT auth guard (rc={rc}): {summary}"], []
-    if "Logged in using ChatGPT" not in stdout:
+    if "Logged in using ChatGPT" not in combined:
         return summary, [f"Codex auth mode requires ChatGPT login; got: {summary}"], []
     return "Logged in using ChatGPT", [], []
 

--- a/tests/test_agent_loop.py
+++ b/tests/test_agent_loop.py
@@ -3775,6 +3775,36 @@ def test_active_codex_runner_execute_fails_closed_without_codex(tmp_path: Path) 
     assert calls == []
 
 
+def test_active_codex_runner_accepts_chatgpt_login_on_stderr(tmp_path: Path) -> None:
+    repo = _write_repo(tmp_path)
+    _write_expanded_active_runner_fixture(repo)
+    calls: list[list[str]] = []
+
+    class DummyProc:
+        stdin = None
+        pid = 7100
+
+        def wait(self, timeout=None):  # type: ignore[no-untyped-def]
+            return 0
+
+    def stderr_chatgpt_auth(cmd, **kwargs):  # type: ignore[no-untyped-def]
+        return subprocess.CompletedProcess(cmd, 0, stdout="", stderr="Logged in using ChatGPT\n")
+
+    result = agent_loop.write_active_codex_runner(
+        execute=True,
+        repo_root=repo,
+        popen_factory=lambda cmd, **kwargs: calls.append(list(cmd)) or DummyProc(),  # type: ignore[arg-type]
+        which_func=lambda exe: "/bin/codex",
+        auth_runner=stderr_chatgpt_auth,
+    )
+
+    assert result.decision == "completed"
+    assert calls
+    state = json.loads(result.state_path.read_text(encoding="utf-8"))
+    assert state["auth_status"] == "Logged in using ChatGPT"
+    assert not any("requires ChatGPT login" in blocker for blocker in result.blockers)
+
+
 def test_active_codex_runner_requires_chatgpt_login_by_default(tmp_path: Path) -> None:
     repo = _write_repo(tmp_path)
     _write_expanded_active_runner_fixture(repo)

